A36-xds-for-servers.md

```diff
@@ -104,13 +104,15 @@ is _not_ percent encoded, as this string is not a URI.
 The xDS-returned Listener must have an [`address`][Listener.address] that
 matches the listening address provided. The Listener's `address` would be a TCP
 `SocketAddress` with matching `address` and `port_value`. The XdsClient must
-NACK the resource if the address does not match.
+NACK the resource if the address does not match. The xDS client must also NACK
+the resource if `Listener.listener_filters` is non-empty.
 
-Although `Filter`s will not be observed initially, the `FilterChain` contains
-data that may be used. When looking for a FilterChain, the standard matching
-logic must be used. Each `filter_chain_match` of the repeated
+Although `FilterChain.filters`s will not be observed initially, the
```

Comment:
Not clear about "observed initially". Are you saying "initially" as in the first few updates from the control plane may not contain `FilterChain.filters` but will be present later? Or "initially" as in the first/initial gRPC implementation might choose to ignore `FilterChain.filters`.

Reply:
Reworded to say "as part of this gRFC"